dtm0/fop.c

```diff
@@ -235,6 +270,11 @@ static void m0_dtm0_send_msg(struct m0_dtm0_service *dtms,
 	req               = m0_fop_data(fop);
 	req->dtr_msg      = msg_type;
 	rc = m0_dtm0_tx_desc_copy(txd, &req->dtr_txr) ?: m0_rpc_post(item);
+
+	phase_sm_id = m0_sm_id_get(&fom->fo_sm_phase);
+	rpc_sm_id   = m0_sm_id_get(&item->ri_sm);
+	M0_ADDB2_ADD(M0_AVI_FOM_TO_TX, phase_sm_id, rpc_sm_id);
```

Comment:
Why do we re-use fom-to-tx here? shouldn't it be something like "fom-to-pmsg"?

Reply:
Let's leave this mapping as is, today, for new scripts mappings are type-less. Eventually we'll remove all typed mappings from the instrumentation schema.